x/wasm/internal/types/types.go

```diff
@@ -6,6 +6,12 @@ import (
 	auth "github.com/cosmos/cosmos-sdk/x/auth/exported"
 )
 
+// Model is a struct that holds a KV pair
+type Model struct {
```

Comment:
This is how I defined it. I will make them both `[]byte` in the next upgrade (very soon), as there was strong desire for binary keys internally by the first contract dev.

Reply:
I agree, binary keys sounds right to me.  It can be casted to string as needed.